main.go

```diff
@@ -478,6 +487,19 @@ func (svr *bridge) handleCall(w http.ResponseWriter, r *http.Request) {
 				}
 			}
 
+			if *clickToGenerator {
+				// sets the notification to be clickable without the need to use
+				// extendedDetails, mainly this is to work with the markdown formatting
+				// so there is no need to add HTML to the notification, and not disturb
+				// the existing flags.
+				if strings.HasPrefix(alert.GeneratorURL, "http") {
```

Comment:
It seems [go requires a nil check here](https://go.dev/play/p/CLzePGOz__d) in the event that the url isn't set on the alert. It might be common for all alertmanagers to do this today, but for futureproofing, a quick nil check is a good idea

Reply:
Oh that's a good call actually I only tested it with my vanilla prometheus one, better be safe than sorry

Reply:
Actually upon double checking [alert.GeneratorURL](https://github.com/DRuggeri/alertmanager_gotify_bridge/blob/master/main.go#L52) is a string so I'll just check for non emptiness